### PR TITLE
Remove async-content placeholder and query the correct element

### LINKED ISSUE #1896: Digital story nav overlaying title
## Type

screen shot 2017-12-04 at 12 06 09

### PR DESCRIPTION
**Pull request by davidpmccormick**
Fixes #1896 

## Screenshot
screen shot 2017-12-11 at 16 51 58

### CODE REVIEW

**Comment by davidpmccormick** [comment]
@wellcometrust/experience-dev here it is without the whitespace changes: https://github.com/wellcometrust/wellcomecollection.org/pull/1922/files?w=1

File: client/js/components/async-content.js [hunk 1]
```diff
@@ -5,46 +5,45 @@ import shrinkStoriesNav from './shrink-stories-nav';
...
-          scrollToClickedItem: false
-        });
+      if (component === 'series-transporter') {
+        const dataModifiers = el.getAttribute('data-modifiers') || JSON.stringify({});
+        const parsedModifiers = JSON.parse(dataModifiers);
+        const modifiers = Object.keys(parsedModifiers)
+          .filter(modifier => modifier)
+          .concat(['transporter', 'in-content']);
+
+        const numberedListTransporter = document.querySelector('.js-numbered-list-transporter');
```
Comment (jamesgorrie) [inline client/js/components/async-content.js]: The reason we can't have this is if we have two async units on the same page, which I know we had somewhere, hence I made the change... I should've left a note.

**Comment by jamesgorrie** [comment]
Saying that - this is a live bug, so it can go in, but can we come up with a solution to both?

**Comment by davidpmccormick** [comment]
Yep, I'd like to solve the underlying issue, but my chops weren't there to do so yesterday. I'll merge this then have another look.

**Pull request merged**